far/FarUkr.hlf.m4

```diff
@@ -5110,17 +5108,21 @@ zero-size expression.
  #{,m}#   - ^<wrap>equals to {0,m}
  #{,m}?#  - ^<wrap>equals to {0,m}?
 
- #"Greedy" and "not greedy" quantifiers#
+ #"Жадібні" та "не жадібний" квантифікатори#
+
+  Жадібний квантифікатор (Greedy) захоплює якомога більше символів і лише якщо
+подальший збіг не вдається, він "повертає" захоплені символи по одному,
+перевіряючи збіг регулярки (такий відкат досить дорогий за ресурсами).
+  Наприклад, під час перевірки, що вираз "A.*Z" відповідає наступному рядку
+"AZXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", вираз #.*# захоплює весь рядок, а потім
+відкочує символ за символом справа наліво, поки не знайде Z, таким чином
+рядок буде перевірено стільки разів, скільки потрібно буде пройти назад на кожну
+літеру X, доки черга не дійде до Z.
 
- Greedy quantifier captures as many characters as possible, and only if
-further match fails, it "returns" the captured string (the rollback
-happens, which is rather expensive).
- When expression "A.*Z" is matched to string
-"AZXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX", #.*# captures the whole string, and then
-rolls back character by character until it finds Z. On the opposite, if the expression
-is "A.*?Z" then Z is found at once. Not greedy quantifier is also known as
-#minimizing#, it captures as few characters as possible, and only if
-further match fails it captures more.
+  На противагу, якщо наш вираз буде "A.*?Z", то Z знайдено відразу, наступним кроком.
+Не жадібний квантифікатор також відомий як #lazy#, захоплює якомога менше символів
+і лише якщо подальший матч не вдається, він йде зліва направо, захоплючи наступний символ,
+та перевіряючи рядок на матчінг.
```

Comment:
Here, English (and other languages') help has two paragraphs, while your translation has three. This causes the paragraph mismatch error:
```
Mismatch: Topics[@RegExp].Paragraphs.Count
    FarEng.hlf.m4: 21
    FarUkr.hlf.m4: 22
```
Please merge two last paragraphs or split this place in other help files.

Reply:
Reformatted to a single paragraph, thanks